### PR TITLE
Update defaults.go

### PR DESCRIPTION
**Pull request by stewart-yu**
**What this PR does / why we need it**:
This PR is a TODO
Add explain about TaintTolerationPriority function.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go [hunk 1]
```diff
@@ -225,7 +225,8 @@ func defaultPriorities() sets.String {
 		// Prioritizes nodes that have labels matching NodeAffinity
 		factory.RegisterPriorityFunction2("NodeAffinityPriority", priorities.CalculateNodeAffinityPriorityMap, priorities.CalculateNodeAffinityPriorityReduce, 1),
 
-		// TODO: explain what it does.
+		// Taints are restrictive markers added to Nodes. Only Pods which explicitly tolerate a particular taint
+		// will be scheduled onto a Node marked with that taint.
```
Comment (resouer) [inline plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go]: It's better to say: `// Prioritizes nodes that ...` and please limit this comment in one sentence and accurate.

**Comment by stewart-yu** [comment]
Thanks. I have modified its explain.

**Comment by stewart-yu** [comment]
@resouer

File: plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go [hunk 2]
```diff
@@ -225,8 +225,7 @@ func defaultPriorities() sets.String {
 		// Prioritizes nodes that have labels matching NodeAffinity
 		factory.RegisterPriorityFunction2("NodeAffinityPriority", priorities.CalculateNodeAffinityPriorityMap, priorities.CalculateNodeAffinityPriorityReduce, 1),
 
-		// Taints are restrictive markers added to Nodes. Only Pods which explicitly tolerate a particular taint
-		// will be scheduled onto a Node marked with that taint.
+		// Prioritizes nodes that marked with particular taint explicitly tolerated by Pods.
```
Comment (zhangxiaoyu-zidif) [inline plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go]: “Prioritizes nodes that marked with taint which pod can tolerate. ” That may be fine.
BTW, squah the commit.

**Review by zhangxiaoyu-zidif — CHANGES_REQUESTED** [review]

File: plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go (hunk 2 above)
Comment (stewart-yu) [reply plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go]: OK. Thanks.

**Comment by resouer** [comment]
ping me back when squashed

**Comment by zhangxiaoyu-zidif** [comment]
process sequence：
1. commit
2. rebase
3. push

**Comment by stewart-yu** [comment]
Commit have  squashed.@resouer

**Pull request merged**